cove_360/tests.py

```diff
@@ -22,7 +25,7 @@
                 'Sponsor(s)': ' ',
                 'Surname of applicant': 'Roe',
                 'amountAwarded': 0,
-                'awardDate': '24/07/2014',
+                'awardDate': '{}-07-24'.format(current_year),
```

Comment:
There's probably a reason why this date was formatted using `DD/MM/YYYY`, so I'm wary of us changing it. @Bjwebb any ideas?

Reply:
Not sure, it could just be bad data copy/pasted here. Since this data is only used to test additional checks, I think the tests would fail if they relied on this.